archive_viewer/table_models/curve_model.py

```diff
@@ -187,3 +225,11 @@ def curve_at_index(self, index: QModelIndex) -> ArchivePlotCurveItem:
             The requested curve.
         """
         return self._plot.curveAtIndex(index)
+
+    @Slot(object)
+    def remove_curve(self, curve: BasePlotCurveItem):
```

Comment:
Can you add a return type hint to the method and expand the docstring to include the input parameters